packages/database/src/migrations/20201116013159-AddMinMaxValuestoMOSReport-modifies-data.js

```diff
@@ -0,0 +1,151 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+  * We receive the dbmigrate dependency from dbmigrate initially.
+  * This enables us to not have to rely on NODE_PATH.
+  */
+exports.setup = function(options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = function(db) {
+  return db.runSql(`
+      UPDATE "dashboardReport"
+      SET "viewJson" = "viewJson" || '{"chartConfig": {
```

Comment:
My personal preference would be to have the chartConfig as a javascript object, and use JSON.stringify() to insert it into the database. This also means that you can make the changes more clear.
```javascript
const OLD_CHART_CONFIG = { ... };

const toInsert = {
   ...OLD_CHART_CONFIG,
   UNFPA_RH_MOS_Condoms_male: {
       ...OLD_CHART_CONFIG.UNFPA_RH_MOS_Condoms_male,
       referenceValue: 12,
    },
   UNFPA_RH_MOS_Condoms_female: {
       ...OLD_CHART_CONFIG.UNFPA_RH_MOS_Condoms_female,
       referenceValue: 6,
    },
}

```